aspnetcore/blazor/fundamentals/static-files.md

```diff
@@ -193,6 +196,62 @@ For examples of how to address the policy violation with Subresource Integrity (
 
 :::moniker-end
 
+:::moniker range=">= aspnetcore-10.0"
+
+### Fingerprint client-side static assets
```

Comment:
Me personal feeling is that this section is hard to understand, it's quite different between Blazor Web App & Blazor WebAssembly Standalone.

So my general question is probably:
Should we mix these modes in a single document?
Or should we focus the document on or the other mode?

I can see to ways of reading docs
- I read it like a book and probably want to learn about the same concepts for both modes
- I have an app and want to solve something, then I would appreciate to select the mode and read the details only for that mode

Mostly just my thoughts :)

Reply:
I'll take a look.

Reply:
I'll break this into two sections on the next commit.